core/src/coproduct.rs

```diff
@@ -781,33 +795,36 @@ pub trait CoproductEmbedder<Out, Indices> {
 }
 
 impl CoproductEmbedder<CNil, HNil> for CNil {
-    fn embed(self) -> CNil
-    { match self {
+    fn embed(self) -> CNil {
+        match self {
         // impossible!
-    }}
+    }
+    }
```

Comment:
Looks like my formatting here confused rustfmt.  I may have to break this habit... (and maybe submit a bug to rustfmt)